core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.html

```diff
@@ -592,6 +592,7 @@ <h3>Canonical Stories</h3>
   .topic-editor .skills-valid {
     background-color: #009C8A33;
     border: 2px solid #009C8A;
+    display: none;
```

Comment:
Hm, if we are going to hide it anyway, the CSS rules above are not going to be used. It can be removed.

Have you considered deleting this class altogether and just using an "ng-if" to show the "Uncategorized Skills" section in the HTML if there are uncategorized skills to show?

Reply:
@kevintab95 made these changes in the latest commit. PTAL.